roles/test_report_send/tasks/reporting/splunk.yml

```diff
@@ -0,0 +1,90 @@
+# SPDX-License-Identifier: Apache-2.0
+---
+# reporting/splunk.yml tasks file for redhatci.ocp.test_report_send
+
+- name: Read content from test report JSON file
+  ansible.builtin.slurp:
+    src: "{{ trs_report_path }}"
+  register: _trs_json_file_content
+
+- name: Decode JSON content of test report file
+  ansible.builtin.set_fact:
+    trs_test_data: "{{ _trs_json_file_content.content | b64decode | from_json }}"
+    trs_default_dt: "{{ now(fmt='%Y-%m-%dT%H:%M:%S') }}"
+
+- name: Print current datetime
+  ansible.builtin.debug:
+    var: trs_default_dt
+
+- name: Setup timestamps broken down to int/float parts for {{ trs_collector_type }}
+  ansible.builtin.set_fact:
+    trs_job_time_int: "{{ (trs_meta_data.metadata.job.created_at | \
+      default(trs_default_dt) | \
+      regex_replace('\\..*$', '') | \
+      to_datetime('%Y-%m-%dT%H:%M:%S')).strftime('%s') | int }}"
+    trs_job_time_fraction: "{{ trs_meta_data.metadata.job.created_at | default(trs_default_dt) | regex_replace('^.*\\.', '0.') | float | round(3) }}"
```

Comment:
Ditto, these vars are only used once, instead use them at the task level as vars